src/ideal/ideal.jl

```diff
@@ -837,7 +837,26 @@ function divrem(I::sideal{S}, G::sideal{S}; complete_reduction::Bool = false) wh
                                                                    false, true, R.ptr)
    libSingular.set_option("OPT_REDSB",old_redsb)
    libSingular.set_option("OPT_REDTAIL",old_redtail)
-   return (smodule{S}(R,ptr_T), sideal{S}(R,ptr_Rest), smodule{S}(R,ptr_U))
+   return (sideal{S}(R,ptr_T), sideal{S}(R,ptr_Rest), smodule{S}(R,ptr_U))
```

Comment:
Maybe you could also add a test for this change in `divrem`.
Moreover, this is a breaking change for OSCAR's implementation of the `reduce_with_...` functionality.